packages/client/test/cli/cli.spec.ts

```diff
@@ -3,19 +3,46 @@ import { Client } from 'jayson/promise'
 import { assert, describe, it } from 'vitest'
 
 describe('[CLI]', () => {
-  it('should start HTTP RPC and return valid responses', async () => {
+  // chain network tests
+  it('should successfully start client with a custom network and network id', async () => {
     const file = require.resolve('../../dist/bin/cli.js')
-    const cliArgs = ['--rpc', '--dev=poa']
+    const cliArgs = ['--network=sepolia', '--networkId=11155111', '--dev=poa']
+    const child = spawn(process.execPath, [file, ...cliArgs])
+    return new Promise((resolve) => {
+      child.stdout.on('data', async (data) => {
+        const message: string = data.toString()
+        if (message.includes('Client started successfully')) {
+          assert.ok('Client started successfully', 'read from HTTP RPC')
+          child.kill()
+          resolve(undefined)
+        }
+        if (message.toLowerCase().includes('error')) {
+          child.kill(9)
+          assert.fail(`client encountered error: ${message}`)
+        }
+      })
+      child.stderr.on('data', (data) => {
+        const message: string = data.toString()
+        assert.fail(`stderr: ${message}`)
+      })
+      child.on('close', (code) => {
+        if (typeof code === 'number' && code > 0) {
+          assert.fail(`child process exited with code ${code}`)
+        }
+      })
+    })
+  }, 18000)
+  // engine rpc tests
+  it('should start engine rpc and provide endpoint', async () => {
+    const file = require.resolve('../../dist/bin/cli.js')
+    const cliArgs = ['--rpcEngine', '--dev=poa']
     const child = spawn(process.execPath, [file, ...cliArgs])
-
     return new Promise((resolve) => {
       child.stdout.on('data', async (data) => {
         const message: string = data.toString()
         if (message.includes('http://')) {
           // if http endpoint startup message detected, call http endpoint with RPC method
-          const client = Client.http({ port: 8545 })
-          const res = await client.request('web3_clientVersion', [], 2.0)
-          assert.ok(res.result.includes('EthereumJS'), 'read from HTTP RPC')
+          assert.ok(message.includes('engine'), 'read from HTTP RPC')
```

Comment:
Same here, let's add an RPC call to an engine endpoint and get a valid response

Reply:
fixed